modules/eco-node-health-check-operator-creating-node-health-check.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-health-check-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-node-health-check-operator-creating-node-health-check_{context}"]
+= Creating a node health check
+Using the web console, you can create a node health check to identify unhealthy nodes and specify the remediation type and strategy to fix them.
+
+.Procedure
+
+. From the *Administrator* perspective of the {product-title} web console, click *Compute* -> *NodeHealthChecks* -> *CreateNodeHealthCheck*.
+. Specify whether to configure the node health check using the *Form view* (the default setting) or the *YAML view*.
+. Enter a *Name* for the node health check. The name must consist of lower case, alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character.
+. Specify the *Remediator* type, *Self node remediation* or *Other*, and the *Remediation strategy*. The Self node remediation strategy applies only to the Self Node Remediation operator. The Self node remediation options are part of the the Self Node Remediation operator that is installed with the Node Health Check operator. Selecting *Other* requires an *API version*, *Kind*, *Name*, and *Namespace* to be entered from an existing resource of a remediator.
```

Comment:
@slintes for version 4.12 will be two Remediation strategies or only the Resource deletion strategy?

Reply:
to my best knowledge it will be resource deletion only. @mshitrit correct?

Reply:
Thanks @anna-savina @slintes. I have left that text as it is as it just asks the user to specify the remediation strategy. It does not list every possible strategy.

Reply:
Works for me. We might get new strategies in future, so this will always be valid 👍🏼